app/controllers/options_controller.rb

```diff
@@ -0,0 +1,25 @@
+class OptionsController < ApplicationController
+  def create
+    selected_option_ids = option_params[:selected_option_ids]
+    cuisine_id = option_params[:cuisine_id]
+    @cuisine = Cuisine.find(cuisine_id)
+    order_item_data = {
+      cuisine: @cuisine,
+      selected_option_ids:
+    }
+    session[:order] = [] if session[:order].nil?
```

Comment:
```suggestion
    session[:order] ||= []
```
khỏi cần if short làm chi em nhé